src/components/taskDetails/Details.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import setColor from './DetailsColor';
+import classNames from './task-details.module.scss';
+
+type Props = {
+  detailType: string;
+  value: string;
+};
+
+const Details: FC<Props> = ({ detailType, value }) => {
```

Comment:
change the component name

Reply:
I think it looks good to me, we can see later if we want to refactor it. Thanks for the suggestion, resolving.